build.sbt

```diff
@@ -0,0 +1,461 @@
+import Dependencies._
+
+lazy val commonSettings = Seq(
```

Comment:
Ok I see now that we moved from a `Build.scala` to a `build.sbt`. Can you convince me this is a good idea?